src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -48,34 +57,37 @@ function isSubscription(item?: SubscriptionTreeItemBase): item is SubscriptionTr
     }
 }
 
-let isVerifyingWorkspace: boolean = false;
-export async function createStaticWebApp(context: IActionContext & Partial<ICreateChildImplContext> & Partial<IStaticWebAppWizardContext>, node?: SubscriptionTreeItemBase): Promise<AppResource> {
+let isVerifyingWorkspace = false;
+export async function createStaticWebApp(context: IActionContext & Partial<ICreateChildImplContext> & Partial<IStaticWebAppWizardContext>, node?: SubscriptionTreeItemBase, _nodes?: SubscriptionTreeItemBase[], ...args: unknown[]): Promise<AppResource> {
+    const isLogicAppParameterPassed = args.length > 0;
+    if (isLogicAppParameterPassed) {
+        type Resource = {backendResourceId: string, region: string, name: string};
+        const logicApp = args[0] as Resource;
+        context.logicApp = logicApp;
+        return await createStaticWebAppWithLogicApp(context);
+    }
+
     if (isVerifyingWorkspace) {
         throw new VerifyingWorkspaceError(context);
     }
-
     const progressOptions: ProgressOptions = {
         location: ProgressLocation.Window,
         title: localize('verifyingWorkspace', 'Verifying workspace...')
     };
-
     isVerifyingWorkspace = true;
     try {
         if (!isSubscription(node)) {
             node = await ext.rgApi.appResourceTree.showTreeItemPicker<SubscriptionTreeItemBase>(SubscriptionTreeItemBase.contextValue, context);
         }
-
         await window.withProgress(progressOptions, async () => {
             const folder = await tryGetWorkspaceFolder(context);
             if (folder) {
                 await telemetryUtils.runWithDurationTelemetry(context, 'tryGetFrameworks', async () => {
                     const detector = new NodeDetector();
-
                     const detectorResult = await detector.detect(folder.uri);
                     // comma separated list of all frameworks detected in this project
                     context.telemetry.properties.detectedFrameworks = `(${detectorResult?.frameworks.map(fi => fi.framework).join('), (')})` ?? 'N/A';
                     context.telemetry.properties.rootHasSrcFolder = (await AzExtFsExtra.pathExists(Uri.joinPath(folder.uri, NodeConstants.srcFolderName))).toString();
```

Comment:
you can see here on the left that you had removed some lines. please add them back